### DISCUSSION TITLE
Assert Array of Objects contains key/value

### DISCUSSION THREAD

**Discussion by hmajid2301** [gavv/httpexpect#126] [Q&A] [answered]
Is there an easy way to assert that an array of objects contains just a specific key.

Let's say I have an object like so:

```json
    {
      "username": "dhawanShikhar",
      "question_pools": [
        {
          "pool_name": "draw_me",
          "game_name": "drawlosseum",
        },
        {
          "pool_name": "my_unique_pool2",
          "game_name": "quibly",
        }
      ]
    }
```

I want to check that at least one of the `pool_name` include a specific value i.e. `my_unqiue_pool2`. In the array of `question_pools`.

**Comment by gavv** [comment, marked as answer]
Hi, I think the closest thing is using JSONPath + Array().Iter() + a loop, see this snippet for details: https://github.com/gavv/httpexpect#json-schema-and-json-path

**Comment by hmajid2301** [comment]
thanks
